.env.example

```
@@ -5,15 +5,17 @@ HOST=localhost
+BASE_OS=alpine
+BASE_VERSION=3.23
+DB_VERSION=17.7
+FORWARD_APP_PORT=3010
 FORWARD_DB_PORT=5432
-STAGING_BRANCH=
 
 # Application Configuration
 APP_NAME=
 APP_VERSION=
-APP_URL="http://localhost:${FORWARD_PORT}"
+APP_URL="http://localhost:${FORWARD_APP_PORT}"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_URL key should go before the APP_VERSION key

(UnorderedKey)